licenses-3rdparties.gradle

```
@@ -114,6 +114,8 @@ downloadLicenses {
     // Add licenses to some libraries which don't declare their license inside the POM. Should be rechecked if the version is updated.
     licenses = [
             'javax.servlet.jsp:jsp-api:2.1' : license('Common Development and Distribution License Version 1.1', null),
+            'org.jline:jline:3.22.0' : license('BSD-2-Clause', "https://opensource.org/license/bsd-2-clause"),
```

```suggestion
            'org.jline:jline:3.22.0' : license('BSD 2-Clause license', 'http://opensource.org/licenses/BSD-2-Clause'),
```

I think this should stop it from adding a "new" whole license clause.

---

Of some reason it is still complaining about invalid license after this change, will take a closer look tomorrow